src/app/protocols/http/v1/schemas/user_schemas.py

```diff
@@ -49,14 +51,25 @@ def validate_authentication(self, data, **kwargs):
                 raise ValidationError('Not a valid string', field_name='authentication')
         else:
         # if using password reset code authentication, authentication field must be an int
-            if type(auth) != int:
-                raise ValidationError('Not a valid integer', field_name='authentication')
+            if type(auth) != str:
```

Comment:
You can remove this validation function and make the auth field a String, since they are both strings now